aip/general/0145.md

```diff
@@ -48,6 +48,25 @@ Exclusive end values are preferable for the following reasons:
 - It is easier to reason about abutting ranges: `[0, x), [x, y), [y, z)`, where
   values are chainable from one range to the next.
 
+### Use of `google.type.Interval`
+
+The [`google.type.Interval`][interval] type (in the common protos described in
+[AIP-213][]) represents a range between two timestamps, with an inclusive start
+value and exclusive end value as described above.
+
+Ranges between two timestamps which conform to the expectations of the
+`Interval` message **should** usually use this rather than having separate
+start and end fields. This allows client code to be written against the
+`Interval` message (such as checking whether a given timestamp occurs within
+the interval) and reused across multiple intervals in the same API, or even
+across multiple APIs.
+
+This is only a weak recommendation, and APIs **may** use start and end
```

Comment:
Nit: Again, I'd prefer we not weaken the guidance beyond the flexibility of the Requirement Words. `However, APIs **may** continue to use...` would be my preference.

Reply:
Let's talk about this in our normal group meeting (this PR isn't urgent). I think there are gradations of advice... some "should" is "you really, really should - it's only not 'must' because there are a few really weird corner cases" whereas this is a pretty mild preference.

Reply:
That's totally fair and yes let's talk about it in our sync.